packages/discord.js/src/structures/interfaces/TextBasedChannel.js

```diff
@@ -135,25 +135,6 @@ class TextBasedChannel {
    * })
    *   .then(console.log)
    *   .catch(console.error);
-   * @example
```

Comment:
Example is shown in guide, so its deleted

Reply:
I don't like the removal of this example, both bc its outside the scope of the PR and also because its a useful example to have

Reply:
then delete sending files?

Reply:
No, why would you? The whole example is useful and correct.

Reply:
https://github.com/discordjs/discord.js/pull/9251#discussion_r1141489850